app/uk/gov/hmrc/helptosavereminder/controllers/HtsUserUpdateController.scala

```diff
@@ -39,15 +39,16 @@ class HtsUserUpdateController @Inject()(
     extends HtsReminderAuth(authConnector, cc) {
 
   def update(): Action[AnyContent] = ggAuthorisedWithNino { implicit request => implicit nino ⇒
-    request.body.asJson.get
+    request.body.asJson
+      .getOrElse(Json.toJson(""))
       .validate[HtsUser]
       .fold(
         error => {
           Logger.error(s"Unable to de-serialise request as a HtsUser: ${error.mkString}")
           Future.successful(BadRequest)
         },
         (htsUser: HtsUser) => {
-          Logger.debug(s"The HtsUser received from frontend to update is : " + htsUser)
+          Logger.debug(s"The HtsUser received from frontend to update is : " + htsUser.nino.nino)
```

Comment:
Shouldn't htsUser.nino.nino be interpolated in the string?  There isn't anything being interpolated

Reply:
Yes, can do.

Reply:
Done